src/makeCalculator.js

```diff
@@ -37,7 +37,48 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    method: '',
+    result: 0,
+    operate(method, number) {
+      calculator.method = method;
+      calculator.result = method(number);
```

Comment:
You are overwriting the result with only the current operation. You should instead apply the operation to the current result.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/call?retiredLocale=uk